src/libertem/udf/feature_vector_maker.py

```diff
@@ -84,10 +85,13 @@ def make_feature_vec(ctx, dataset, num, delta, center=None, rad_in=None, rad_out
         masked_sstd = sstd*mask
     else:
         masked_sstd = sstd
-    coordinates = peak_local_max(masked_sstd, num_peaks=num, min_distance=0)
+    if not (min_dist is None):
```

Comment:
Better to read: `if min_dist is not None:`. Another possibility would be to have only a single `peak_local_max` call, and set `min_dist` to 1 if it is `None`.